packages/snaps-utils/src/manifest/validation.test.ts

```diff
@@ -216,6 +230,16 @@ describe('createSnapManifest', () => {
     expect(() => createSnapManifest(getSnapManifest())).not.toThrow();
   });
 
+  it('throws an error for an invalid dynamic permissions specification', () => {
+    const snapManifest = getSnapManifest();
+    // Duplicate permissions
+    snapManifest.dynamicPermissions = MOCK_INITIAL_PERMISSIONS;
```

Comment:
```suggestion
    const snapManifest = getSnapManifest({ dynamicPermissions: MOCK_INITIAL_PERMISSIONS });
```

Can't we do something like this instead?

Reply:
Yes, updated.